### ISSUE TITLE
Don't show rta if check_ping is not able to reach the host

### ISSUE THREAD

**Issue by RincewindsHat** [monitoring-plugins/monitoring-plugins#1696]
Pinging an not existing host like this:
```
./plugins/check_ping -H 192.168.0.123 -w 60,1% -c 100,5%
PING CRITICAL - Packet loss = 100%|rta=100.000000ms;60.000000;100.000000;0.000000 pl=100%;1;5;0
```
shows round trip average time altough this is wrong in this case and will produce wrong numbers.

**Comment by RincewindsHat** [comment]
#1697

**Comment by RincewindsHat** [comment]
fixed with #1697
